frontend/src/features/apartment/ApartmentSalesPage.tsx

```diff
@@ -118,6 +118,14 @@ const LoadedApartmentSalesPage = ({
                 });
             }
             if (!data.ownerships.length) {
+                ctx.addIssue({
+                    code: z.ZodIssueCode.custom,
+                    message: errorMessages.ownershipPercent,
```

Comment:
The error messages seem to be the wrong way around. Now the `ownershipPercent` message gets shown if there is no owners - and `noOwnerships` if the total percentage isn't 100.